api/tests/functional-tests/backend/metrics/test_text_generation.py

```diff
@@ -0,0 +1,957 @@
+from unittest.mock import patch
+
+import pytest
+from sqlalchemy.orm import Session
+
+from valor_api import crud, enums, schemas
+from valor_api.backend import models
+from valor_api.backend.core import create_or_get_evaluations
+from valor_api.backend.metrics.text_generation import (
+    _calculate_rouge_scores,
+    _calculate_sentence_bleu,
+    _compute_text_generation_metrics,
+    compute_text_generation_metrics,
+)
+
+# Moved PREDICTIONS here so that they are accessible by mock functions.
+PREDICTIONS = [
```

Comment:
nit: should PREDICTIONS and REFERENCES be fixtures (similar to text_generation_test_data)? they'd still be accessible by mock functions

Reply:
When I change predictions to a fixture (as a function), it is accessible for the tests and other fixtures, but not for the mock functions. I'm adding predictions as a parameter of the mock functions. Is there another way to do this?

Reply:
I changed references to a fixture, but left PREDICTIONS as a module level variable, as it doesn't work as a fixture with mock functions.